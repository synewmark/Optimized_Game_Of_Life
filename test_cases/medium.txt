version https://git-lfs.github.com/spec/v1
oid sha256:9846cde2f493c6a4046df19d4d60ebce5ffbe765e556de4630ad6453af215b5d
size 754
