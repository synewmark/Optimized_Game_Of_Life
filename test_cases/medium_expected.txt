version https://git-lfs.github.com/spec/v1
oid sha256:e9e5b25dc8bd98dc3ce1da1a90d3cece430978eb6d0783519dedd829f46e519b
size 365
